packages/calcite-components/src/components/table-header/table-header.scss

```diff
@@ -3,12 +3,11 @@
  *
  * These properties can be overridden using the component's tag as selector.
  *
- * @prop --calcite-table-header-background: Specifies the background color of the component.
- * @prop --calcite-table-header-background: [Deprecated] Use the `--calcite-table-header-background-color` property instead. Specifies the background color of the component.
- * @prop --calcite-table-header-background-color: Specifies the background color of the component.
- * @prop --calcite-table-header-border-color: Specifies the border color of the component.
- * @prop --calcite-table-header-heading-color: Specifies the heading color of the component.
- * @prop --calcite-table-header-description-color: Specifies the description color of the component.
+ * @prop --calcite-table-header-background: [Deprecated] Use the `--calcite-table-header-background-color` property instead. Specifies the component's background color.
+ * @prop --calcite-table-header-background-color: Specifies the component's background color.
+ * @prop --calcite-table-header-border-color: Specifies the component's border color.
+ * @prop --calcite-table-header-heading-text-color: Specifies the component's `heading` color.
```

Comment:
This is missing `text` in the description:

```
Specifies the component's `heading` text color.
```

Applies to description text color prop.

Reply:
Added, thanks